frank.go

```diff
@@ -130,8 +135,16 @@ func boot() {
 }
 
 func main() {
-	listenersReset()
 	setupFlags()
+
+	if *listenHttp != "" {
+		go func() {
+			log.Fatal(http.ListenAndServe(*listenHttp, nil))
```

Comment:
The HTTP server will — in practice — only return an error when it can’t bind the listening port. This is a case where we would indeed want to loudly complain.